benchmark/server/main.go

```diff
@@ -16,6 +16,16 @@
  *
  */
 
+/*
+Package main provides a server used for benchmarking.  It launches a server
+which is listening on port 50051. An example to start the server would be to
```

Comment:
You have double space after the sentences for the rest of the PR.
```suggestion
which is listening on port 50051.  An example to start the server would be to
```